docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
```

Comment:
`pip` at least version eight, but I think that's it.

Anything else, @soltanmm? I'm reluctant to say `virtualenv` without converting the whole tutorial to be `virtualenv`-based.